packages/babel-preset-babili/src/index.js

```diff
@@ -1,20 +1,50 @@
+const err = new Error("Babili requires babel-core>=6.14.0. Upgrade babel-core or tools dependent on babel-core to recent versions");
+
+function isVersion(versionStr) {
+  const version = versionStr.split(".").map((p) => parseInt(p));
+  return version[0] >= 6 && version[1] >= 14;
```

Comment:
this is reverted. because it seems not possible to check version in preset. 

But yeah, cool.. it should be `===6`